checkout/v3/index.md

```diff
@@ -1,48 +1,125 @@
+or two if that makes more sense for your business.
 
-[https]: /introduction#connection-and-protocol
+**Consumer Info:**  The consumer's personal data e.g. name, address, phone
+number etc.
+
+**Authendication:** The process for verifying the consumer's identity via
+Strong Consumer Authentication (e.g. BankID).
+
+**Delivery Info:** Information about where the goods should be delivered.
+
+**PSP:** The service of providing payment methods in the checkout or payment
+menu.
+
+## What Are You Looking For?
+
+**Full Checkout**
+**Payments Only**
+
+## Full Checkout
+
+By using the Full Checkout, we help you collect and safely store consumer data.
+We can also prefill consumer info in the checkout if they have agreed to let us
+store their info. All of our implementations support both single payment options
+and the full payment offering.
+
+**Standard**
+
+We collect and verify the identity of your consumer. We also collect the billing
+and shipping address, and we store the consumer information. With our PSP you
+are always able to choose one or more payment methods.
+
+Data Ownership          Swedbank Pay                  Merchant side
+
+Authentication          v
+Delivery Info           v
+Consumer Info           v
+PSP                     v
+
+{% include iterator.html next_href="/checkout/v3/standard/setup/"
+                         next_title="Proceed" %}
+
+**Authenticated**
+
+The consumer data you have collected, is shared with us for verification and
+storage. If you have a login, your consumer won't need to fill out their
+information twice. With our PSP you are always able to choose one or more
+payment methods.
+
+Data Ownership          Swedbank Pay                  Merchant side
+
+Authentication          v
+Delivery Info           v
+Consumer Info                                         v
+PSP                     v
+
+{% include iterator.html next_href="/checkout/v3/authenticated/setup/"
+                         next_title="Proceed" %}
+
+**Merchant Authenticated Consumer**
+
+In order to implement this solution, you need to be able to both collect and
+verify your consumer's data using SCA. We store the consumer information, and
+with our PSP you are always able to choose one or more payment methods.
+
+Data Ownership          Swedbank Pay                  Merchant side
+
+Authentication                                        v
+Delivery Info                                         v
+Consumer Info           v
+PSP                     v
+
+{% include iterator.html next_href="/checkout/v3/mac/setup/"
+                         next_title="Proceed" %}
+
+## Payments Only
+
+If you are looking for our payments package, you will have the flexibility to
+build your own checkout flow. You collect the consumer data and own the entire
+checkout process. The **Payments Only** implementation supports both single
+payment options and the full payment offering.
+
+**Payments**
```

Comment:
remark-lint is reacting to this line where emphasis is being used instead of a heading. 

```suggestion
### Payments
```